NOTICE

```diff
@@ -1,5 +1,5 @@
 Apache Drill
-Copyright 2013-2022 The Apache Software Foundation
+Copyright 2013-2023 The Apache Software Foundation
```

Comment:
Looks like this PR should be rebased on the latest master. Probably these changes are present because of the force push to master.

Reply:
Fixed.